Software/target/stm32wlxx_hal_conf.h

```diff
@@ -125,7 +125,7 @@
   * @brief Internal Low Speed oscillator (LSI) value.
   */
 #if !defined (LSI_VALUE)
-#define LSI_VALUE                           32000UL     /*!< LSI Typical Value in Hz*/
+#define LSI_VALUE                           250UL       /*!< LSI Typical Value in Hz*/

```

Comment:
As we discussed, it doesn't matter if you have max iwdg of 66 mins, it will need to be kicked in the end periodically by it's own function, or scheduled task, so the flexibility is good but when needed and now it's not needed